src/ESPAsyncWebServer.h

```diff
@@ -43,6 +43,8 @@
   #error Platform not supported
 #endif
 
+#include "literals.h"
```

Comment:
IMO this is OK to include it in CPP files, but not in headers otherwise this thing will leak to the user code and I really don't even want to think about that or the possible breakage / collisions that might happen.